Docs migrations

lego963 opened the pull request:
Resolve: #577

lego963 committed:
Docs migrations initial commit

gtema reviewed (changes requested):
Please go through the readme and fix links to old repo location (unfortunately GH does not allow commenting lines not touched by PR)

lego963 committed:
Formatted all resources

lego963 committed:
Formatted all data-sources

gtema commented:
you should not remove the "--- frontmatter ---" headers from files. At least page_title should stay (actually perhaps start using subcategory). Read carefully https://www.terraform.io/docs/registry/providers/docs.html

lego963 commented:
Yes, I will return it, now I format the documents and fix them with strange or insufficient info

lego963 committed:
Update .gitignore

Remove redundant job from zuul

Remove unused funcs in Makefile

Change golang package to opentelekomcloud

Add new layout for data sources

lego963 committed:
Reformat resources

Add subcategory to resources

outcatcher commented on docs/data-sources/antiddos_v1.md:
```diff
@@ -1,63 +1,59 @@
-
-* `trigger_http_pps` - The HTTP request rate at the triggering point.
+---
+subcategory: "AntiDDos"
```
It should be `DDoS`, not `DDos`

gtema reviewed (dismissed):
Whew.
I don't know whether we will get problems with removal of `title`, but we can re-add it then.

lego963 commented:
>`page_title - The title of this document, which will display in the docs navigation. This is only required for documents in the guides/ folder.`

according to https://www.terraform.io/docs/registry/providers/docs.html

gtema commented:
> > `page_title - The title of this document, which will display in the docs navigation. This is only required for documents in the guides/ folder.`
> 
> according to https://www.terraform.io/docs/registry/providers/docs.html

it's not about "requires", it's about how the page name would be shown by the render tool, but even before it was not very much useful. On the other hand all other providers do prefix pages with the provider name (like it it currently)

gtema commented:
hmm, the build is also failing in a weird way

lego963 committed:
Add subcategory for data sources

lego963 committed:
Fix issues with subcategoies

lego963 committed:
Revert misstake changes in vendor

lego963 committed:
remove line between brackets

lego963 committed:
Clean up examples folder

lego963 committed:
remove lines

gtema reviewed (approved):
No clue how to review 271 files other that simply by diagonal
